support-frontend/assets/images/favicons.ts

```diff
@@ -9,3 +9,5 @@ require('./favicons/120x120.png');
 require('./favicons/144x144.png');
 
 require('./favicons/152x152.png');
+
+export {};
```

Comment:
This file doesn't have any exports as it's an entrypoint specified in `webpack.entryPoints.js`. Adding this empty export makes typescript happy and hasn't affected the build